Utils/Extensions/NSURLExtensions.swift

```diff
@@ -96,6 +96,24 @@ extension NSURL {
         }
         return nil
     }
+    
+    public func absoluteStringWithoutProtocol() -> String? {
```

Comment:
Rather than doing this, let's go in the other direction: compute the string we want.
For example, for 

```
http://m.bbc.com/news/world-us-canada-33767775?foo=bar
```

we'd want

```
bbc.com/news/world-us-canada-33767775
```

That's something like

```
public func baseDomainAndPath() -> String {
    return self.baseDomain() + (self.path ?? "/")
}
```